### ISSUE TITLE
Plugin build from Git incompatible with 2017.1

### ISSUE THREAD

**Issue by russel** [kingsleyh/DLanguage#139]
I build the plugin from my repository clone and seemed to get a zipfile with 1.12-SNAPSHOT, but when I tried to load this into IDEA 2017.1 it rejects it with the message:
```
Plugin 'D Language' is incompatible with this installation
```
which is a bit of a blocker on me trying this out.

(I want to avoid using the downloaded 2016.3.)

**Comment by SingingBush** [comment]
not started looking at 2017.1 compatibility yet. You wouldn't need to manually download an earlier version of intellij. The Gradle plugin handles all that so it's just a case of setting the `ideaVersion` in gradle.properties to an Inteliij version you want to start running the plugin in such as `171-EAP-SNAPSHOT`. Then just `./gradlew runIdea` from the root of the project.

For when we start focusing on 2017.* versions it'll probably be best to edit the `plugin.xml` to have the idea version as being since 171:

```xml
<idea-version since-build="171"/>
```

I've done this locally and found there was a minor issue in `DLanguageLibraryType.class` that needed fixing and it seems to not work with Gradle 2.13 so I updated the `gradle-wrapper.properties` to use v2.14 and it now boots up.

I've pushed the changes to the develop branch. see commit: https://github.com/kingsleyh/DLanguage/commit/55c0963efecdb197bba1a85a7e3dacad1ce3605b
